website/docs/foundations/focus-ring/index.md

```diff
@@ -1,52 +1,59 @@
 
 ```css
 .your-selector {
   [...your CSS declarations]
   &:focus,
   &:focus-visible {
     box-shadow: var(--token-focus-ring-action-box-shadow);
+    border-radius: inherit;
   }
 }
 ```
 
-!!! Critical
-
-*   The design token as CSS variable can be used **only** with a `box-shadow` property.
-*   The border radius depends on the UI element to which it is applied, so it’s up to you to apply the correct `border-radius`. (Tip: Consider using the `inherit` value.)
-!!!
-
 ### CSS helper classes
 
-We provide a **CSS helper class** `.hds-focus-ring-box-shadow`, that is a wrapper of the design token. It’s unlikely you can use it directly in a template because this style is connected to the “focused” pseudo-state of an element (more likely it would be used in composition with other classes).
+The CSS helper class `.hds-focus-ring-box-shadow` is a wrapper of the design token. It’s likely you’ll need to use this helper class in composition with other classes (instead of directly in a template) because it’s connected to the element’s “focused” pseudo-state.
 
-To use this class you have to import the CSS file `[products|devdot]/css/helpers/focus-ring.css` from the `@hashicorp/design-system-tokens` package.
+To use this class, import the CSS file:
 
-!!! Critical
+```scss
+// for product applications
+@import "~@hashicorp/design-system-tokens/dist/products/css/helpers/focus-ring.css";
 
-The border radius depends on the UI element to which it is applied to, so it’s up to you to apply the right `border-radius` (tip: consider to use the `inherit` value).
-!!!
+// for hashicorp developer platform
+@import "~@hashicorp/design-system-tokens/dist/devdot/css/helpers/focus-ring.css";
+```
 
 ### Sass mixins
 
-We have created two **Sass mixins** `hds-focus-ring-basic` and `hds-focus-ring-with-pseudo-element`, but they’re mainly used for internal use (to the design system codebase). These mixins do more than just apply the focus style: they also take care of all the different way to declare the `:focus/:focus-visible` for different browsers.
+While we offer two Sass mixins `hds-focus-ring-basic` and `hds-focus-ring-with-pseudo-element`, they’re mainly used in the design system codebase. 
 
-To use these mixins you have to import the Sass file `packages/components/app/styles/mixins/_focus-ring.scss` contained in the `@hashicorp/design-system` monorepo or the same file `app/styles/mixins/_focus-ring.scss` distributed in the `@hashicorp/design-system-components` package.
+In addition to applying the focus style, these mixins also account for all declarations of `:focus/:focus-visible` in the different browsers.
+
+If needing to use these mixins, import the Sass file: 
+
+<!-- IS THIS CORRECT? ARE THERE OTHER DETAILS WE NEED TO INCLUDE? -->
+```scss
+@import "~@hashicorp/design-system-components/app/styles/mixins/_focus-ring.scss";
```

Comment:
If I just copy&paste this, it is very likely that it will not work (I have to do other things, for the build process to reference that file correctly). That's why I kept it "generic".

Reply:
The way all of our Foundational documentation is written assumes that one has gone through and set things up appropriately based on the README or the new "Getting started for engineers" content. This was the case even before I made this change. I can include a link to the "Getting started" content on each page so that it's clear there are more steps involved before one gets to this point, but what was there originally was hard to follow and not very actionable, so I'm trying to improve it. 

I hoping to understand if this is the appropriate path and way to import this file into one's project. And if there are other steps related to **mixins** specifically that we need to include (as there is no documentation on mixins in the Getting started content). 

Happy to jump on a call to understand this process more too.